functional-tests/src/test/groovy/specs/FlowSpecs.groovy

```diff
@@ -0,0 +1,66 @@
+package specs
+
+import pages.HomePage
+import pages.InformationPage
+import pages.RegistryPage
+import pages.SubmissionsPage
+import pages.ExternalLinkPage
+
+import spock.lang.Timeout
+import spock.lang.Title
+import spock.lang.Narrative
+import spock.lang.Unroll
+
+@Timeout(60)
+@Title('Groundwater wells Flow Tests')
+@Narrative('''
+As a developer, I want to ensure all page links work, have the correct text, and direct to the correct page.
+''')
+class FlowSpecs extends LoggedInSpec {
+
+  void setup() {
+    logInAsAdminUser()
+  }
+
+  @Unroll
+  void 'Navigate Page from: HomePage, click header Link: #TextSelector, Assert Page: #AssertPage'() {
+    given: 'I start on the HomePage'
+      to HomePage
+    when: 'I click on the header link with label: #TextSelector.text'
+      headerModule.clickMenuItem(TextSelector)
+    then: 'I arrive on the #AssertPage.getSimpleName()'
+      at AssertPage
+    where:
+      TextSelector                       || AssertPage
+      [ text:'Well Search' ]             || HomePage
+      [ text:'Groundwater Information' ] || InformationPage
+      [ text:'Registry' ]                || RegistryPage
+      [ text:'Submit Report' ]           || SubmissionsPage
+  }
+
+  @Unroll
+  void 'Navigate Page from: HomePage, click footer Link: #TextSelector, Assert Page: #AssertPage'() {
+    given: 'I start on the HomePage'
+      to HomePage
+    when: 'I click on the footer link with label: #TextSelector.text'
+      footerModule.clickMenuItem(TextSelector)
+    then: 'I arrive on the #AssertPage.getSimpleName()'
+      at AssertPage
+    where:
+      TextSelector               || AssertPage
+      [ text:'Home' ]            || new ExternalLinkPage('B\\.C\\. Homepage - Province of British Columbia',
```

Comment:
I question value of these tests. It is non-critical functionality, in code (a footer), that should very rarely (if ever) change. I think this should be deferred to manual testing.